scanner/templates/laravel/.fly/entrypoint.sh

```diff
@@ -12,5 +12,12 @@ else
     /usr/bin/php /var/www/html/artisan view:cache --no-ansi -q
     chown -R webuser:webgroup /var/www/html
 
+    ## Run user scripts, if they exist
+    for f in /var/www/html/.fly/scripts/*.sh; do
```

Comment:
Nice, should cache preparation go to `.fly/scripts`?

Reply:
Yep, altho I may tweak this some more so be able to hook into key points:

1. When the Docker image is being built
2. When a container spins up (called from the `ENTRYPOINT`)
3. **Can you think of anything else?**

The main things to things to think about in terms of how Fly works and what Laravel needs for deployments (migrations, setting caches, etc):

1. The `docker build` doesn't have access to env vars / build secrets without extra work
2. `release_command` commands run in a separate VM just before deployments ([reference](https://fly.io/docs/reference/configuration/#the-deploy-section)) 
    - Doesn't have access to mounted volumes
    - Any file-system changes are lost (e.g. laravel view cache files). This means the `artisan view:cache` and other commands should run somewhere else, generally
    - Has access to the network, so migrations against databases running separately should be fine (but a local sqlite database change is not fine, see point 2)